packages/webapp-libs/webapp-api-client/src/graphql/apolloClient.ts

```diff
@@ -19,20 +18,6 @@ const IS_LOCAL_ENV = ENV.ENVIRONMENT_NAME === 'local';
 
 export const emitter = new Emitter();
 
-export const subscriptionClient = createClient({
-  url: ENV.SUBSCRIPTIONS_URL,
```

Comment:
I don't think we are using this env anymore? Would need to remove it from `webapp/env.shared`, docs etc